scripts/zulip_emoji_reactions.py

```diff
@@ -131,16 +147,16 @@ def add_reaction(name: str, emoji_name: str) -> None:
         # If the emoji is a custom emoji, add the fields `emoji_code` and `reaction_type` as well.
         print("Removing previous reactions, if present.")
         if has_peace_sign:
-            remove_reaction('delegated', 'peace_sign')
+            remove_reaction('delegated', 'peace_sign', 'delegated')
         if has_bors:
-            remove_reaction("bors", "bors", emoji_code="22134", reaction_type="realm_emoji")
+            remove_reaction("bors", "bors", "ready-to-merge", emoji_code="22134", reaction_type="realm_emoji")
         if has_merge:
-            remove_reaction('merge', 'merge')
+            remove_reaction('merge', 'merge', "")
```

Comment:
Why is there an empty label here and below?

Also, I know that this is preexisting, but do you know why `merge` is repeated?